client/signup/steps/site-or-domain/style.scss

```diff
@@ -39,7 +39,7 @@
 	text-align: center;
 
 	.button {
-		text-transform: uppercase;
+		text-transform: none;
```

Comment:
I think we should just remove this line altogether since the default `text-transform` is none anyway. And then I guess we can simplify the `&.site-or-domain__is-placeholder` selector too since there is no need to have an empty `.button` selector section, right?

Reply:
Thanks for taking a look, @hambai! These are great suggestions. I shall get on this at the earliest.